src/main.cpp

```diff
@@ -420,11 +422,31 @@ int main(int argc, char* argv[])
      * does not mention this particular restriction it is not unreasonable to
      * assume the same condition applies...
      */
-    QString homeLink = QStringLiteral("%1/mudlet-data.lnk").arg(QDir::homePath());
+    QString homeLinkWindows = QStringLiteral("%1/mudlet-data.lnk").arg(QDir::homePath());
+    QFile oldLinkFile(homeLink);
+    if (oldLinkFile.exists()) {
+        // A One-time fix up past error that did not include the ".lnk" extension
+        oldLinkFile.rename(homeLinkWindows);
+        QMessageBox::information(nullptr,
```

Comment:
It's not something the user needs to know about - lets print it to STDOUT instead. I can guarantee you that 99% of Windows users will have no idea what this message will be about and at most will just get confused and aliented by it :wink:

Reply:
Printing to STDOUT DOES NOT WORK ON A WINDOWS GUI APPLICATION!!! 😢

{So `mudlet /v` or `mudlet /?` does not produce anything that the user can see on that platform - in the medium term I will revert to the `-v`/`--version`/`-h`/`--help` as that is consistent to the other _built-in_ Qt options; also I will provide the "console" output on a `QMessageBox` for just THAT platform - that __is__ an approach that I have seen in other Windows Applications!}

Reply:
Oh `mudlet /q` DOES work...